ArticleTemplates/assets/scss/modules/_related-content.scss

```diff
@@ -9,4 +9,8 @@
         width: 1200px;
     }
  }
- 
\ No newline at end of file
+
+ .is_advertising.garnett--type-guardianlabs[data-content-type='gallery'] .related-content {
+    background: color(global-adv-bg);
```

Comment:
I was thinking of making the "paid for by" section this colour and keeping related content how it is?

Reply:
oh sorry, I will change that, i must have misunderstood on the ticket

Reply:
<img src="https://user-images.githubusercontent.com/19835654/63359152-25cf4f80-c364-11e9-84fa-a7b51898ba3c.png" width="300px" /> - now looks like this